client/src/pages/Plan/Year.js

```diff
@@ -115,7 +115,11 @@ const Year = ({ year, data, startYear, updateData, courses, setStartYear }) => {
     return (
         <Flex direction="column" mt={4}>
             <Flex width="100%" justify="center" align="center" marginTop="20px">
-                <InlineEdit title="Year" value={startYear + year} onChange={(e) => setStartYear(e - year)} />
+                <InlineEdit
+                    title="Year"
+                    value={startYear + year}
+                    onChange={(e) => new RegExp("^[0-9]+$").test(e) && setStartYear(e - year)}
```

Comment:
Beautiful line of code

Reply:
gotta prevent the user from entering letters :p